Documentation/Settings.cfg

```diff
@@ -1,41 +1,16 @@
 # coding: utf-8
 
-# #####
```

Comment:
Changes like these are out of scope of the PR

Reply:
A separate commit should be enough, not?